src/wiktextract/extractor/simple/text_utils.py

```diff
@@ -0,0 +1,17 @@
+import re
+
+from .section_titles import POS_DATA
+
+# A regex for detecting a heading that starts like a POS: "Noun 1"
+POS_STARTS_RE = re.compile(
```

Comment:
I think this long regex could also be removed.

IMO the code extract other sections like gloss lists and sounds have more room to be further simplified, still fell too complicated for the simple layout used in this edition...

Reply:
I've removed it and replaced it with a "detect last digits" regex and checking the start against POS_HEADINGS, I'm just working on some stuff involving it so I'll commit later.